build-system/common/utils.js

```diff
@@ -145,6 +145,10 @@ function getFilesToCheck(globs, options = {}, ignoreFile = undefined) {
       log(green('INFO: ') + 'No files to check in this PR');
       return [];
     }
+    // forbidden-terms always needs to be checked.
+    if (!filesChanged.includes('build-system/test-configs/forbidden-terms.js')) {
+      filesChanged.push('build-system/test-configs/forbidden-terms.js');
+    }
     return logFiles(filesChanged);
   }
   return ignored.filter(globby.sync(globs, options));
```

Comment:
In addition to the change to `utils.js`, I think we should update the `LINT_RULES` build target so that the entire repo is linted during CI if any of our custom lint rules are changed.

Suggested edit can be found at https://github.com/rsimha/amphtml/commit/8da50dc8778409118c85b3b3c378e16d74d7cbcd#diff-19e974deb1e4e50db022e0a4c6b920052ff4bb12bf4382375fa2387aa012104cR235-R243.